nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/MergeRecord.java

```diff
@@ -89,6 +89,7 @@
     @WritesAttribute(attribute = "merge.count", description = "The number of FlowFiles that were merged into this bundle"),
     @WritesAttribute(attribute = "merge.bin.age", description = "The age of the bin, in milliseconds, when it was merged and output. Effectively "
         + "this is the greatest amount of time that any FlowFile in this bundle remained waiting in this processor before it was output"),
+    @WritesAttribute(attribute = "merge.uuid", description = "UUID of the merged flow file that will be added to the original flow files attributes"),
```

Comment:
Pure nitpick: "will be added to the original _FlowFile's attributes_" and I also prefer `FlowFile` instead of `flow file` just to be consistent across NiFi world. Feel free to ignore though :)

Reply:
You're right, always good to ensure consistency.